### PR TITLE
fix(tianmu): fix stonedb has two LZ4 versions (#471)

### LINKED ISSUE #471: bug: stonedb has two LZ4 versions
### Describe the problem

1. in extra/lz4/lz4-1.9.3, there are two files lz4.cpp lz4.h.(obviously, the version is 1.9.3)
2. in stonedb\storage\tianmu\compress, there are two files lz4.cp lz4.h(from lz4.h,we can see that the version is 1.7.1)
3. two version’s lz4 is used，though the tianmu's lz4 using namespace Tianmu::compress, but some abnoraml behavier can be found.
3.1 firstly, the objcect file lz4.cpp.o( tianmu file) 's symbol(using nm command) ,function LZ4_uncompress is 
 _ZN6Tianmu8compress14LZ4_uncompressEPKcPci; while function LZ4_compress is LZ4_compress( no namespace name).
though, stonedb5.7 can be compiled pass，but maybe exists some risk.
3.2 in stondb8.0,in  ubuntu  the gcc report multiple definition described as below:(centos7.9 works well)
      https://github.com/andyli029/mysql-server-mysql-8.0.30/issues/102
3.3 in stondb8.0,delete the tianmu's  lz4.cpp lz4.h files, it works well, the build system can find the extra's lz4.
4. anyway,it is not necessary to use two different lz4 version, delete is and depend on the extra is a suitable option(tianmu's verison is older).

### Expected behavior

_No response_

### How To Reproduce

_No response_

### Environment

version:StoneDB5.7/StoneDB8.0  
OS:UBUNTU20.04

### Are you interested in submitting a PR to solve the problem?

- [X] Yes, I will!

### PR DESCRIPTION
**Pull request by lujiashun**
(1). delete lz4-1.9.3;
(2). add lz4-1.9.4;
(3). delete tianmu’s lz4 files lz4.cpp，lz4.h;

## Summary about this PR

Issue Number: close #471

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

## Changelog

- [ ] New Feature
- [X] Bug Fix
- [X] Improvement
- [X] Performance Improvement
- [ ] Build/Testing/CI/CD
- [ ] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: cmake/lz4.cmake [hunk 1]
```diff
@@ -34,7 +34,7 @@ MACRO (FIND_SYSTEM_LZ4)
   ENDIF()
 ENDMACRO()
 
-SET(LZ4_VERSION "lz4-1.9.3")
```
Comment (RingsC) [inline cmake/lz4.cmake:37]: Pls, following the commit guide.   such as, PR title, commit message, etc.

File: storage/tianmu/compress/text_compressor.cpp [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
 #include "common/assert.h"
 #include "compress/inc_wgraph.h"
-#include "compress/lz4.h"
```
Comment (RingsC) [inline storage/tianmu/compress/text_compressor.cpp:25]: Using clang-format to formatting these line codes.

**Review by RingsC — CHANGES_REQUESTED** [review]

**Review by RingsC — APPROVED** [review]
LGTM

**Review by isredstar — APPROVED** [review]
LGTM

**Pull request merged**